tests/experimental/starry/vis_test.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+
+from jaxoplanet.experimental.starry.orbit import SurfaceBody
+from jaxoplanet.experimental.starry.surface import Surface
+from jaxoplanet.experimental.starry.visualization import show_map
+from jaxoplanet.experimental.starry.ylm import Ylm
+
+
+def test_show_map():
```

Comment:
It looks like we'll need to install `matplotlib` for this test to work.

Reply:
I think we could skip it if `matplotlib` isn't installed, because that is tested in the `comparison` tests (`starry` requires `matplotlib`). Or is that a wrong practice to rely on other test environments?

Reply:
Sure that would be fine! We should add a `pytest.importorskip` in that case then.